spec/controllers/api/public/v1/courses_controller_spec.rb

```diff
@@ -202,6 +202,19 @@
           )
         end
       end
+
+      context "courses count" do
```

Comment:
@gpeng - noticed that there are no request specs, only controller specs. I've followed convention in this case but wondering this and some of the other specs in this file should be moved to a request spec?

Reply:
Yes. Not sure what happened there. We do 'do' request specs generally. I'll get them moved if we ever have a lull/firebreak. This is good for now though.